### PR TITLE
chore(other): adjust docs for deployment, neo4j cypher commands, and have new scripts

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Adjust docs for deployment, neo4j cypher commands, and have new scripts

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

**Review by Tirokk — DISMISSED** [review]

File: deployment/scripts/cluster.dashboard-tunnel.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```
Comment (roschaefer) [inline deployment/scripts/cluster.dashboard-tunnel.sh:1]: I would say just use `k9s` and drop the kubernetes dashboard. Or use both and use `k9s` to set up the tunnel for you.

File: deployment/scripts/cluster.backend-bash.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  echo "!!! You must provide a CONFIGURATION via environment variable !!!"
+  exit 1
+fi
+
+printf "  Cluster:  %s\n" $CONFIGURATION
+
+# configuration
+KUBECONFIG=${KUBECONFIG:-${SCRIPT_DIR}/../configurations/${CONFIGURATION}/kubeconfig.yaml}
+
+kubectl --kubeconfig=${KUBECONFIG} -n default exec -it $(kubectl --kubeconfig=${KUBECONFIG} -n default get pods | grep ocelot-backend | awk '{ print $1 }') -- /bin/sh
```
Comment (roschaefer) [inline deployment/scripts/cluster.backend-bash.sh:22]: This is to SSH into the backend? It's easier accomplished with `k9s`.

**Review by roschaefer — CHANGES_REQUESTED** [review]
I would say let's remove the bash scripts, no?

File: deployment/scripts/cluster.dashboard-tunnel.sh (hunk 1 above)
Comment (Tirokk) [reply deployment/scripts/cluster.dashboard-tunnel.sh:1]: Thanks for the hint @roschaefer !
I get your point and we will change this in the future. 💫

Please just leave it that way for now and we will modernize all these in the next step.
Cool?

File: deployment/scripts/cluster.backend-bash.sh (hunk 2 above)
Comment (Tirokk) [reply deployment/scripts/cluster.backend-bash.sh:22]: Yes, this is to SSH into the backend.

same same as obove?

File: deployment/scripts/cluster.dashboard-tunnel.sh (hunk 1 above)
Comment (roschaefer) [reply deployment/scripts/cluster.dashboard-tunnel.sh:1]: If you just need an approval, sure, here you go!

**Review by roschaefer — APPROVED** [review]

File: deployment/scripts/cluster.backend-bash.sh (hunk 2 above)
Comment (roschaefer) [reply deployment/scripts/cluster.backend-bash.sh:22]: You can do port-forwarding, SSH, logs and a lot more with `k9s`. I guess the features of `k9s` all cover what the bash scripts are trying to accomplish.

**Pull request merged**